tutorials-v4/time-evolution/006_photon_birth_death.md

```diff
@@ -0,0 +1,130 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Monte Carlo Solver: Birth and Death of Photons in a Cavity
+
+<!-- #region -->
+Authors: J.R. Johansson and P.D. Nation
+
+Modifications: C. Staufenbiel (2022)
+
+### Introduction
+
+In this tutorial we demonstrate the *Monte Carlo Solver* functionality implemented in `qutip.mcsolve()`. For more information on the *MC Solver* refer to the [QuTiP documentation](https://qutip.org/docs/latest/guide/dynamics/dynamics-monte.html). 
+
+We aim to reproduce the experimental results from:
+
+
+
+>  Gleyzes et al., "Quantum jumps of light recording the birth and death of a photon in a cavity", [Nature **446**,297 (2007)](http://dx.doi.org/10.1038/nature05589).
+
+
+In particular, we will simulate the creation and annihilation of photons inside an optical cavity due to the thermal environment when the initial cavity is a single-photon Fock state $ |1\rangle$, as presented in Fig. 3 from the article.
```

Comment:
some of the latex doesn't render, maybe it's just an issue with markdown?

Reply:
For me all LaTeX in this notebook renders fine. I'd suggest merging this PR and check what nbviewer generates?